docs/api-docs/storefront/graphql/graphql-api-overview.md

```diff
@@ -95,10 +90,10 @@ To explore the storefront GraphQL schema, checkout the **Docs** and **Schema** t
 
 ## Authentication
 
-GraphQL Storefront API requests are authenticated with tokens sent via the HTTP `Authorization` header.
+GraphQL Storefront API requests are authenticated with tokens sent using the HTTP `Authorization` header.
 
 
-```bash
+```shell title="Example Authorization header using cURL"

```

Comment:
```suggestion
```shell title="Example authorization header using cURL"

```

Reply:
@bc-andreadao I'm treating `Authorization` as the proper name of the `Authorization` header because it's a standard header, not just a BC one.